R/endpoints.R

```diff
@@ -1,16 +1,20 @@
-#' fetch AFHSB data (point data, no min/max)
+#' Fetch AFHSB data (point data, no min/max)
 #'
-#' API docs: https://cmu-delphi.github.io/delphi-epidata/api/afhsb.html
+#' @param auth Character. Authentication token.
+#' @param locations Character vector. Locations to fetch (region/state, or 3-letter country code labels).
+#' @param epiweeks Epirange. Epiweeks to fetch in the form epirange(startweek, endweek), where
+#'   startweek and endweek are in the form YYYYWW.
+#' @param flu_types Character. Flu types to fetch (disjoint (flu1, flu2-flu1, flu3-flu2, ili-flu3)
```

Comment:
suggestion: clarify: character vector, or string?  Same for future instances of "Character."

Reply:
suggestion: put quotes around the options, e.g., `"flu2-flu1"`

Reply:
Removing this, so we don't duplicate information in API docs and here.